### PR TITLE
Fix typos on documentation and code

### PR DESCRIPTION
**Pull request by electrocucaracha**

### CODE REVIEW

File: controllers/specializer-operator/blueprint/specialzer-operator/Kptfile
Comment (johnbelamaric) [inline controllers/specializer-operator/blueprint/specialzer-operator/Kptfile:1]: The directory name will need to change too, right?

File: krm-functions/nad-fn/testdata/golden/create_full_resources/interface-n3.yaml [hunk 1]
```diff
@@ -8,5 +8,5 @@ spec:
   networkInstance:
     name: vpc-ran
   cniType: sriov
-  attachementType: vlan
+  attachmentType: vlan
```
Comment (johnbelamaric) [inline krm-functions/nad-fn/testdata/golden/create_full_resources/interface-n3.yaml:11]: Do the Go type and CRD for these (if there is one) all have the right name?

**Review by johnbelamaric — COMMENTED** [review]
Wow! Lots of updates. A couple questions.

File: krm-functions/nad-fn/testdata/golden/create_full_resources/interface-n3.yaml (hunk 1 above)
Comment (gvbalaji) [reply krm-functions/nad-fn/testdata/golden/create_full_resources/interface-n3.yaml:11]: I think it's spelled correctly there.
https://github.com/nephio-project/api/blob/main/nf_requirements/v1alpha1/interface_types.go

**Comment by johnbelamaric** [comment]
I am going to merge this, but please send a follow up PR with the rename of blueprint/specialzer-blueprint to fix it.

/approve
/lgtm

File: controllers/specializer-operator/blueprint/specialzer-operator/Kptfile
Comment (electrocucaracha) [reply controllers/specializer-operator/blueprint/specialzer-operator/Kptfile:1]: You're right, it needs to change it

**Comment by electrocucaracha** [comment]
> I am going to merge this, but please send a follow up PR with the rename of blueprint/specialzer-blueprint to fix it.

And maybe this will require some changes on the image build process as well

\cc @radoslawc

**Pull request merged**